datanode/gateway/graphql/schema.graphql

```diff
@@ -3943,25 +3943,25 @@ type EthCallTrigger {
 union TriggerKind = EthTimeTrigger
 
 type EthTimeTrigger {
-  Initial: Timestamp
-  Every: Timestamp
-  Until: Timestamp
+  initial: Timestamp
+  every: Int
+  until: Timestamp
 }
 
 type Normaliser {
-  Name: String!
-  Expression: String!
+  name: String!
+  expression: String!
 }
 
 type EthCallSpec {
-  Address: String!
-  Abi: [String!]
-  Method: String!
-  Args: [String!]
-  Trigger: EthCallTrigger!
-  RequiredConfirmations: Int!
-  Normalisers: [Normaliser!]
-  Filters: [Filter!]
+  address: String!
+  abi: [String!]
+  method: String!
+  args: [String!]
+  trigger: EthCallTrigger!
+  requiredConfirmations: Int!
+  normalisers: [Normaliser!]
+  filters: [Filter!]
 }
```

Comment:
Could we have some docs comments here?

Reply:
Yes, added comments here - https://github.com/vegaprotocol/vega/pull/9425/commits/a5577253a18cb1d634b248c803109afe8ea6bef5
Thanks!